pkg/cmd/cli/cmd/deploy.go

```diff
@@ -126,6 +127,7 @@ func NewCmdDeploy(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.C
 	cmd.Flags().BoolVar(&options.cancelDeploy, "cancel", false, "Cancel the in-progress deployment.")
 	cmd.Flags().BoolVar(&options.enableTriggers, "enable-triggers", false, "Enables all image triggers for the deployment config.")
 	cmd.Flags().BoolVar(&options.follow, "follow", false, "Follow the logs of a deployment")
+	cmd.Flags().MarkDeprecated("cancel", "use oc rollout cancel instead")
```

Comment:
Move it below the flag. Can you also deprecate in a separate commit `enableTriggers` in favor of `oc set triggers`?